### DISCUSSION TITLE
Pip install not working

### DISCUSSION THREAD

**Discussion by bathientran** [supabase/supabase#3550] [Questions] [answered]
Hi everyone!

I'm trying to install the supabase-py package using `pip install supabase` but it throws the following error:
```
ERROR: Could not find a version that satisfies the requirement supabase (from versions: none)
ERROR: No matching distribution found for supabase
```
Anyone got a solution for this? When I use `pip3` it works, however, I'm deploying my app on Heroku and there, Heroku uses `pip` and not `pip3` which fails the installation of the packages.

**Comment by J0** [comment, marked as answer]
Hey,

One of the supabase-py Maintainers here:

Supabase-py only supports `python3` 

`pip3` primarily uses Python3 while `pip` is a soft-link to a python installer which is decided based on your $PATH (this can be the Py2 `pip` or `pip3`)

Therefore, in order to fix this we would need to figure out where `pip` points to by doing `which pip` and thereafter ensure that it points to a `python3` installer


Note that as of yesterday, Heroku has deprecated its free tier -- if you'd like to migrate over to Supabase do check out our [migration guide](https://www.google.com/search?client=safari&rls=en&q=migrate+from+heroku+to+supabase&ie=UTF-8&oe=UTF-8)
